Jenkinsfile

```
@@ -120,6 +120,6 @@ def notifyComplete(String buildStatus) {
     else if (buildStatus == SUCCESS && prevBuildResult != SUCCESS)
         slackSend (
             color: '#00FF00', // green
-            channel: '#rev-ai-dev', 
+            channel: '#rev-ai-dev',
```

Please change this as per the notes on the issue
```suggestion
            channel: '#revai-alerts-nonprod',
```

---

Do the same with the failure message

---

Please use channelID (`CFPMB0BK4`) instead of channel name. Make sure to leave a comment describing that that the channel Id refers to #revai-alerts-nonprod.

ChannelId does not change when we rename the slack channel